packages/core/package.json

```diff
@@ -27,5 +27,11 @@
     "mini-css-extract-plugin": "^2.4.2",
     "webpack": "^5.56.0",
     "webpack-cli": "^4.8.0"
-  }
+  },
+  "peerDependencies": {
+    "@department-of-veterans-affairs/formation": "*",
+    "react": "^16.3.1",
```

Comment:
I made a typo here. I meant to use `^16.13.1` like we already have in `react-components`:

https://github.com/department-of-veterans-affairs/component-library/blob/c145ac3802840c39f0b6e94b14c94146829a3a90/packages/react-components/package.json#L99

This is also why I didn't immediately use react 17, but we've already been using the `react-components` subpackage in `vets-website` for a while now with react 17 with no known problems.